mappings/DocumentMapping.json

```diff
@@ -1,16 +1,61 @@
 {
+  "settings": {
+    "analysis": {
+      "analyzer": {
+        "simple": {
+          "type": "custom",
+          "tokenizer": "standard",
+          "filter": [
+            "lowercase"
+          ]
+        },
+        "tex_analyzer": {
+          "type": "custom",
+          "tokenizer": "tex_tokenizer"
+        }
+      },
+      "tokenizer": {
+        "tex_tokenizer": {
+          "type": "pattern",
+          "pattern": "\\$([^\\$]+)\\$",
```

Comment:
Looking at this together: if we understand correctly, this will match on double-dollar-sign-enclosed TeXisms -- basically ignoring the outer $. Right?

Reply:
Should we match one or more $ at the start and end? E.g. ``[\\$]+([^\\$]+)[\\$]+``